frontend/packages/shared/src/types/api/AddLayoutSetResponse.ts

```diff
@@ -0,0 +1,6 @@
+import type { LayoutSetConfig, LayoutSets } from 'app-shared/types/api/LayoutSetsResponse';
+import type { KeyValuePairs } from 'app-shared/types/KeyValuePairs';
+
+export type ResponseObject = LayoutSets | KeyValuePairs;
```

Comment:
Maybe we should refactor this to ```LayoutSetsResponse``` as ```ResponseObject```  is not easy to understand which response or object it refers to? 😄